lib/Controller/PageController.php

```diff
@@ -47,12 +48,14 @@ class PageController extends Controller {
 	public function __construct($appName,
 								IRequest $request,
 								IEventDispatcher $eventDispatcher,
+								IConfig $config,
 								IInitialStateService $initialStateService) {
-		parent::__construct($appName, $request);
+		parent::__construct($appName, $request, $config);
```

Comment:
This is not needed here. :)
The construct of the parent (Controller, see line 38) doesn't needs $config

Reply:
https://github.com/nextcloud/server/blob/master/lib/public/AppFramework/Controller.php#L74